### PR TITLE
Migrate terraform provider to use latest vpc-go-sdk v1.0.0

### PR DESCRIPTION
**Pull request by astha-jain**

### CODE REVIEW

File: ibm/resource_ibm_is_vpc_routing_table_route.go [hunk 1]
```diff
@@ -159,7 +159,7 @@ func resourceIBMISVPCRoutingTableRouteCreate(d *schema.ResourceData, meta interf
 		Address: core.StringPtr(nextHop),
```
Comment (hkantare) [inline ibm/resource_ibm_is_vpc_routing_table_route.go:159]: I think we need to handle both address and id ..something simialr to cli code
https://github.ibm.com/Bluemix/riaas-cli/blob/4c716cc0c529a4192961263ec7cf9ea501582b8d/plugin/commands/vpc/route-table-route.go#L333

Comment (astha-jain) [reply ibm/resource_ibm_is_vpc_routing_table_route.go:159]: This issue is in progress in a separate branch.

File: ibm/resource_ibm_is_instance_group_manager.go [hunk 2]
```diff
@@ -291,7 +292,8 @@ func resourceIBMISInstanceGroupManagerRead(d *schema.ResourceData, meta interfac
 		ID:              &instanceGroupManagerID,
 		InstanceGroupID: &instanceGroupID,
 	}
-	instanceGroupManager, response, err := sess.GetInstanceGroupManager(&getInstanceGroupManagerOptions)
+	instanceGroupManagerIntf, response, err := sess.GetInstanceGroupManager(&getInstanceGroupManagerOptions)
```
[the remark this answers is not in the document]
Comment (astha-jain) [reply ibm/resource_ibm_is_instance_group_manager.go:295]: Let's say it is being changed in preparation for an upcoming feature. I'll make the change in the test files.

[the remark this answers is not in the document]
Comment (astha-jain) [reply ibm/resource_ibm_is_instance_group_manager.go:295]: I am able to build the provider without any issues and Tests using GetInstanceGroupManager looks like following 
```
		_, _, err := sess.GetInstanceGroupManager(&getInstanceGroupManagerOptions)
		if err == nil {
			return fmt.Errorf("instance group manager still exists: %s", rs.Primary.ID)
		}
```

**Pull request merged**